db/migrate/20160303184449_change_test_results_table.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeTestResultsTable < ActiveRecord::Migration
+  def change
+    rename_column :test_results, :input_description, :input
+    change_column_null :test_results, :input, true
```

Comment:
I think we should leave these columns as non-null, and set a default value of the empty string instead.